tasks.py

```diff
@@ -0,0 +1,83 @@
+from robocorp.tasks import task
+from robocorp import browser, http, excel
+from RPA.PDF import PDF
+
+@task
+def robot_spare_bin_python():
+    """Robot to enter weekly sales data into the RobotSpareBin Industries Intranet."""
+
+    # Without this, nothing is visible as it is running headless
+    browser.configure(
+        slowmo=100,
+        browser_engine="chromium",
```

Comment:
And this is the default, so it can be omitted

Reply:
Done